components/_common_/Button/__tests__/Button.test.js

```diff
@@ -13,18 +13,28 @@ describe('Button', () => {
 
   test('should render properly with some props assigned', () => {
     createSnapshotTest(
-      <Button
-        theme="secondary"
-        data-custom-attr="custom stuff here"
-        disabled
-        fullWidth
-        type="submit"
-      >
+      <Button theme="secondary" disabled fullWidth type="submit">
         Test
       </Button>,
     );
   });
 
+  test('should spread data and aria props', () => {
+    createSnapshotTest(
+      <Button aria-label="test" data-attr="test">
+        Test
+      </Button>,
+    );
+
+    const wrapper = shallow(
```

Comment:
I see what you were saying with alongside.

i think in this case, we could just drop the extra snapshot test and only . test for lines 29-36 while _keeping_ the test name you have